static/compiler.js

```diff
@@ -659,6 +683,29 @@ define(function (require) {
             this.astViewOpen = false;
         }
     };
+
+    Compiler.prototype.onRtlPassSelected = function (id, pass) {
+        if (this.id == id) {
```

Comment:
We are trying to move away from unintended type coercion as much as possible. Change this accordingly please :)

Reply:
Could you elaborate this ? I'm not sure I understand what you expect. I've copy/pasted form the AST/CFG code, but maybe their style is also subject to change.

Reply:
Yes, it is. We have had issues on the past where strings were used instead of numbers, that's why, I think, we allow for tyoe coercion in so many places. That should now be fixed tho, so no need for it anymore.

Im just simply asking to use `===` instead of `==` :)